examples/ibm-catalog/standard-solution/versions.tf

```diff
@@ -21,7 +21,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.44.3"
+      version = "= 1.44.3"
```

Comment:
I guess this was done to workaround the known issue in 1.45.0? Can you please create an internal GIT issue to track changing this back to a range when the provider fix is in?